lib/reporters/html.js

```diff
@@ -35,12 +35,12 @@ exports = module.exports = HTML;
 /**
  * Stats template.
  */
-
+var statsBar = '<h1 id="progress-bar"></h1>';
```

Comment:
Also there is already a progress bar. See the `li class="progress"`. I feel this new bar isn quite noisy visually for what it does and think a green/red indicator would be better in the current bar.

Reply:
@craigtaub the reason I went with creating a new bar was because the original issue opened was talking about not wanting to waste eye movement to look towards the count in the upper right hand corner. My thinking was that just coloring the progress circle would still be "using eye movement" to confirm there being an error.
```
to make it more obvious without wasting eye-movement effort to look at the count haha. I prefer github.com/visionmedia/mocha-matrix for this reason but it's not a good fit as a default
```

Also their example in mocha-matrix has a similar visual representation of test results displayed front and center.
https://github.com/tj/mocha-matrix

Reply:
@craigtaub 
I will review the updating to see why it stayed green with 2 errors.

As for why I went with the bar was because the original open issue asked for a way to display test failures without wasting eye movement, and the mocha-matrix project they referenced has a similar visual representation of when tests don't succeed.
```
to make it more obvious without wasting eye-movement effort to look at the count haha. I prefer github.com/visionmedia/mocha-matrix for this reason but it's not a good fit as a default
```

https://github.com/tj/mocha-matrix